argoverse/utils/make_att_files.py

```diff
@@ -0,0 +1,380 @@
+import sys
```

Comment:
@alliecc thanks for adding this, Allie! I think the `sys` import is unused, can we remove it? Also can we order the imports in the following order:

1. Python builtins
(space)
2. external libraries
(space)
3. then argoverse modules?